### DISCUSSION TITLE
Lock Screen Widget - No Seek Bar

### DISCUSSION THREAD

**Discussion by nikorc** [Goodwy/PlayBooks#52] [Ideas]
Requesting a way to remove the seek bar from the lock screen widget. I have an issue where it get touch on accident moving my progress to a later part in the audio book.

Thanks
Nic

**Comment by intrepidclass** [comment]
Seconded! I would also remove the previous and next buttons, and leave only the 20 seconds back button and the pause button.

I would also like a lock option for the main player. To temporarily disable the seek bar.
Or an option to go back to the previous location when you've just accidentally moved the seek bar.
